app/views/admin/users/_user_results.html.haml

```diff
@@ -0,0 +1,26 @@
+= simple_form_for :search, method: :get do |s|
+  = s.input :search_term, label: "User email search", placeholder: "type email here", input_html: { value: (params[:search][:search_term] unless params[:search].blank?) }
+  = s.button :submit, "SEARCH"
+
+- if @users.present?
+  %table
+    %colgroup
+      %col{style: 'width: 80%'}
+      %col{style: 'width: 20%'}
+    %thead
+      %tr
+        %th= sortable_heading_for(:email)
+        %th.actions Actions
+    %tbody
+      - @users.each do |user|
+        %tr
+          %td= user.email
+          %td.actions
+            - if policy(user).edit?
+              = link_to "Edit", edit_url.call(user), class: "btn btn-small"
+            - if policy(user).destroy?
+              = link_to "Delete", delete_url.call(user), method: :delete, data: { confirm: "Are you sure you want to delete #{user}?" }, class: "btn btn-small"
+  = paginate @users
+
+- else
+  %p= "No users matched that search"
```

Comment:
I'd prefer it if the `else` code here had the same HTML as the `@users.present?` code, so that they both result in a table that looks the same, as opposed to just a `<p>`.
![screen shot 2015-11-12 at 11 39 33](https://cloud.githubusercontent.com/assets/4482567/11110025/244a8546-8932-11e5-9272-5ee8c6090aac.png)